src/gui/guiBackgroundImage.h

```diff
@@ -0,0 +1,37 @@
+/*
+Part of Minetest
```

Comment:
Why `part of`? Other files just have `Minetest`.

Reply:
Afaik, it means that the file is part of minetest.

See also https://github.com/minetest/minetest/pull/8740#discussion_r312014639.

Reply:
All files are part of Minetest tho. Also, the linked comment says nothing about adding "Part of"